WOXML.js

```diff
@@ -0,0 +1,142 @@
+var util = require("./util");
+var builder = require("xmlbuilder");
+
+module.exports = class WOXML {
+    constructor(data, agencyMap) {
+        this.root = builder.create("WONetworkDeal");
+        this.data = data;
+        this.agencyMap = agencyMap;
+    }
+
+    getXML() {
+        return this.root;
+    }
+
+    buildHeader() {
+        var header = this.root.ele("Header")
+            .ele("ChangeStatus", "New Deal").up()
+            .ele("Property", "AMC").up() //Network?
+            .ele("DealStatusIncomplete", "Planning Costs").up() //blind copy from example
+            .ele("DealStatusWorking", "Working").up()           //blind copy from example
+            .ele("TotalAmount", this.data.extras.headerTotalAmount).up()
+            .ele("TotalEQUnits", this.data.extras.adSize.value / 30).up() //Equiv 30s
+            .ele("TotalImps", 0).up();
+
+        var headerAdvertiser = header.ele("Advertiser")
+            .ele("Name", this.data.campaign.name).up()
+            .ele("Advertiser", "!PLACEHOLDER!").up()
+            .ele("DealType", "MassEx").up()
+            .ele("DealYear", this.data.extras.dealYear).up()
+            .ele("Transaction", "Cash").up()    //blind copy from example
+            .ele("Marketplace", "Scatter").up() //blind copy from example
+            .ele("RateCard", "AMC Mass Ex").up()
+            .ele("RatingStream", "C3").up()     //blind copy from example
+
+        var headerAdvertiserDemos = headerAdvertiser.ele("Demos")
+            .ele("Demo")
+            .ele("Name", "HH").up()             //blind copy from example (excel says "HH for now")
+            .ele("Primary", "Yes").up()         //blind copy from example
+            .ele("PostingExport", "Yes").up()   //blind copy from example
+            
+        headerAdvertiser.ele("Guaranteed", "No").up()
+            .ele("PriorityCode", "MassEx").up()
+            .ele("ExternalID", this.data.match.id).up() //"There is an ExternalID tag that can be used for [our matched order ID]"
+            .ele("PriorityCode", "MassEx").up()
+            .ele("Equivalized", "Yes").up()
+
+        var agency = header.ele("Agency")
+            .ele("Primary", `${this.data.match.buy.mp.name} (${this.agencyMap[this.data.match.buy.mp.name]})`);
+
+        var sales = header.ele("Sales")
+            .ele("AccountExecutives")
+            .ele("AccountExecutive")
+            .ele("Name", "Mass Exchange").up()  //Excel says "AE" in "AMC Master List"
+            .ele("Primary", "Yes").up()         //blind copy from example
+            .ele("Percent", 100);               //blind copy from example
+
+        return this;
+    };
+
+    buildFlights() {
+        var flights = this.root.ele("Flights")
+            .ele("Flight")
+            .ele("Start", this.data.extras.quarter.start).up()
+            .ele("End", this.data.extras.quarter.end);
+            //.ele("Active", "Yes");
+
+        return this;
+    }
+
+    buildAdvBrands() {
```

Comment:
is this func just fully unused? if so why not comment out the whole func so it just can't be called? (I am assuming it will be important later but it is not know exactly why or how yet)

Reply:
It already doesn't do anything so didn't see the harm in leaving it there. If it gets called it has no effect, but once we do have something to put in there it'll already be called in places that use it.